CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Enhancements
 
+* Added support for inverse relationships which can be declared by the `@LinkingObjects` annotation (#607).
```

Comment:
Perhaps rephrase it to mention that then? At least I was confused about this wording, and I would still add a mention about the queries. Something like:

```
* Support for inverse relationships declared by the @LinkingObjects annotation is now considered stable.
* Added support for queries across reverse relationships (#607).
```